app/code/Magento/Customer/Block/Account/AuthenticationPopup.php

```diff
@@ -105,10 +105,20 @@ public function getBaseUrl()
 
     /**
      * Get customer register url
-     *
+     * @deprecated
+     * @see getCustomerRegisterUrl
      * @return string
      */
     public function getCustomerRegisterUrlUrl()
+    {
+        return $this->getCustomerRegisterUrl();
+    }
+
+    /**
+     * Get customer register url
+     * @return string
+     */
+    private function getCustomerRegisterUrl()
```

Comment:
I think this one should be `public` instead of `private`, because the original method was also public?